src/main/java/org/spongepowered/api/text/Text.java

```diff
@@ -933,6 +936,37 @@ public static Text of(Object... objects) {
         return builder.build();
     }
 
+    public static Optional<Text> get(Dictionary dictionary, String key, Locale locale) {
```

Comment:
IMO it will just confusion if there is for example `Text.of(String)` and `Text.get(String)` even though both methods are **completely** different.